gtnh-modpack.json

```diff
@@ -296,7 +296,7 @@
       "filename": "catwalks-1.7.10-2.1.3-GTNH.jar",
       "download_url": "https://api.github.com/repos/GTNewHorizons/Catwalks-2/releases/assets/58081296",
       "browser_download_url": "https://github.com/GTNewHorizons/Catwalks-2/releases/download/2.1.3-GTNH/catwalks-1.7.10-2.1.3-GTNH.jar",
-      "license": "explicit permission (german)",
+      "license": "Explicit Permission (german).",
```

Comment:
Could you delete the "german" part here? The actual permission in the relevant screenshot is not in German.